Sources/DataConnect.swift

```diff
@@ -14,9 +14,9 @@
 
 import Foundation
 
-import FirebaseAppCheck
-import FirebaseAuth
-import FirebaseCore
+@preconcurrency import FirebaseAppCheck
+@preconcurrency import FirebaseAuth
+@preconcurrency import FirebaseCore
```

Comment:
I'm gradually working through adding Swift 6 conformance to the SDKs, I'll prioritize addressing these ones so we can remove the preconcurrency attribute.

Reply:
No worries - thanks!